### DISCUSSION TITLE
What is "archarm_latest"

### DISCUSSION THREAD

**Discussion by conacts** [uraimo/run-on-arch-action#133] [Q&A] [answered]
Is `archarm_latest` an arch linux distribution?

**Comment by uraimo** [comment, marked as answer]
Hi, yes, it's a user provided aarch64/arm64 docker image since the Arch project only provides amd64 images.
You can see it here: https://github.com/uraimo/run-on-arch-action/blob/master/Dockerfiles/Dockerfile.aarch64.archarm_latest
At the time there were some discussions on this and I expected that aarch64 would have been supported in the official channel at some point leveraging the work done on this image, it seems that this didn't happen and there hasn't been any update in 2 years.
Note that considering that we now have the `base_image` option, this is a prime candidate for removal in one of the next releases.
